mne/preprocessing/tests/test_maxwell.py

```diff
@@ -0,0 +1,134 @@
+# Author: Mark Wronkiewicz <wronk@uw.edu>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import warnings
+import numpy as np
+from numpy.testing import (assert_equal, assert_allclose,
+                           assert_array_almost_equal)
+from nose.tools import assert_true, assert_raises
+
+from mne.preprocessing import maxwell
+from mne.datasets import testing
+from mne.io import Raw, proc_history
+from mne.utils import slow_test, _TempDir
+warnings.simplefilter('always')  # Always throw warnings
+
+
+def test_spherical_harmonic():
+    """Test for spherical harmonics"""
+    from scipy.special import sph_harm as scipy_sph_harm
+
+    deg = 1
+    order = -1
+    azimuth = np.random.random_sample(size=(50, 1)) * 2 * np.pi
+    polar = np.random.random_sample(size=(50, 1)) * np.pi
+
+    # Internal calculation
+    sph_harmonic = maxwell._sph_harmonic(deg, order, azimuth, polar)
+    # Check against scipy
+    sph_harmonic_scipy = scipy_sph_harm(order, deg, azimuth, polar)
+
+    assert_array_almost_equal(sph_harmonic, sph_harmonic_scipy, decimal=15,
+                              err_msg='Spherical harmonic mismatch')
```

Comment:
The restarting didn't help because it ran the same set of values...

If you're really curious about the bug, then you could print out the values of `azimuth` and `polar` before doing the a`assert_array_almost_equal`, my guess is it is some particular combination (maybe both very close to zero?) causing the problem.